file_transaction_service.py

```diff
@@ -0,0 +1,104 @@
+import grpc
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+import server
+import utility
+
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+    """A FileTransactionService class contains services for handling file transactions."""
+
+    def __init__(self, server):
+        self.__server = server
```

Comment:
We are passing the `Server` itself to `FileTransactionService` to persist the `state`